### ISSUE TITLE
Aggregator with Disclosure doesn't auto focus to the new disclosure element

### ISSUE THREAD

**Issue by JakeLaCombe** [cerner/terra-framework#260]
# Bug Report

## Description
When using keyboard to go through the aggregator example at https://engineering.cerner.com/terra-framework/#/raw/tests/terra-aggregator/aggregator/aggregator-with-disclosure, the focus doesn't automatically go to the new disclosure, making it difficult to use the new disclosure with a keyboard input. It seems as though focus should go straight to the new disclosure that opened.

## Steps to Reproduce

1. Go to https://engineering.cerner.com/terra-framework/#/raw/tests/terra-aggregator/aggregator/aggregator-with-disclosure
2. Use the tab to select one of the items
3. Press Enter or Space
4. Press Tab.

**Comment by bjankord** [comment]
This is reproducible using VoiceOver on iOS as well. VO will read when row in section is selected, but never shifts focus to new disclosure. If user taps on new disclosure, VO on iOS will start reading content the user interacted with in the disclosure. This makes it difficult for users to know a new content disclosure is available if they are unable to see the screen.

**Comment by supreethmr** [comment]
I Re-created the Issue on both Browser and Screen reader for Voice over, focus was not moving to disclosure manager on Tab key press. 

This happens when aggregator has more than one item like shown in the example 'Combined Disclosure Aggregator- side panel disclosure' which has multiple sections. Since disclosure-manager is not presented through Aggregator unlike the modal disclosure manager focus does not moves to disclosure manager from Aggregator. Instead of moving focus to side-panel disclosure focus shifts to next item on aggregator list on Tab key press. since aggregator and disclosure are two different components focus shifts to next item on the aggregator list on Tab key press and not shifting focus from aggregator to disclosure in the [example](https://engineering.cerner.com/terra-ui/#/components/terra-aggregator/aggregator/aggregator)

It seems to be focus request should be made explicitly to aggregator on Tab key Press to move Focus from aggregator to disclosure manager. Incase Modal Disclosure Manager Focus will be moved to disclosure on-Open by default since it will be presented as modal and need not to make request to get focus. 

Focus can only be requested through aggregator-delegate which can be set while implementing disclosure manager with aggregator and Since Disclosure is treated more like Div and not like element in Dom I was not able to set focus to disclosure container on Tab key press on aggregator item. 

I would like to know if there are any possible work-around which can be implemented to shift focus from Aggregator item to side panel disclosure manger on Tab key Press.

**Comment by ryanthemanuel** [comment]
Closing in lieu of https://github.com/cerner/terra-framework/issues/321
